mkdocs.yml

```diff
@@ -47,23 +48,26 @@ nav:
         - Logging: admin/collecting-logs/README.md
         - Metrics: admin/collecting-metrics/README.md
         - Uninstalling Knative: admin/install/uninstall.md
+    # Developer guide
     - Developer guide:
         - Overview: developer/README.md
         - Knative Serving:
           - Services:
             - About Knative Services: developer/serving/services/README.md
             - Creating a Service: developer/serving/services/creating-services.md
-            - Configure resource requests and limits: developer/serving/services/configure-requests-limits-services.md
             - Configuring private Services: developer/serving/services/private-services.md
-    - Serving Component:
+            - Configuring custom domains: developer/serving/services/custom-domains.md
+            - Configure resource requests and limits: developer/serving/services/configure-requests-limits-services.md
+          - Troubleshooting:
+            - Debugging application issues: developer/serving/troubleshooting/debugging-application-issues.md
+    # Serving
+    - Knative Serving:
```

Comment:
We could drop `Knative` from `Knative Serving` if we need space to fit more top level items.
Same for the guides it can be "Administration" "Development"

Reply:
Let's not do that please, I think we need "guide" for context and as part of our attempt to make docs more consistent we should be using Knative Serving and Knative Eventing throughout - the comments here are just for my own reference to help me find things in the long-winded yml file